test/tests.py

```diff
@@ -2052,6 +2052,7 @@ def test(self):
             print("dml %s;" % (self.dml_version,), file=f)
             print("device test;", file=f)
             print("/// COMPILE-ONLY", file=f)
+            print("/// DMLC-FLAG --no-compat=broken_unused_types", file=f)
```

Comment:
You need? Huh.... I thought with dml-api on 7 the ordering of types/symbols in the txt file happened to favor us (and its only with your experimentation that things got upset.)